titan-server/src/cors_fairing.rs

```diff
@@ -0,0 +1,25 @@
+use rocket::http::Method;
+use rocket::{get, routes};
+use rocket_cors::{AllowedOrigins, Cors};
+
+pub fn cors() -> Cors {
+    // @todo move allowed origins to config
```

Comment:
I agree with this, though, I don't think it's a big deal. In local development, this is an issue. In production, we'll be working off the same domain, so CORS, in theory, shouldn't be an issue.

We actually can use the webpack dev proxy to proxy the frontend and server through the same ports so CORS isn't a concern in dev. Just a thought. We can discuss it more tonight.